docs/fastdds/dds_layer/core/status/status.rst

```diff
@@ -3,3 +3,630 @@
+
+* **total_count**: Total cumulative count of missed deadlines for any instance read by the
+  current :ref:`dds_layer_subscriber_dataReader`.
+  As the deadline period applies to each instance of the :ref:`dds_layer_topic_topic` independently,
+  the count will will be incremented by one for each instance for which data
+  was not received in the deadline period.
+
+* **total_count_change**: The change in **total_count** since
+  the last time |on_requested_deadline_missed| was called or the status was read.
+  It can only have zero or positive values.
+
+* **last_instance_handle**: Handle to the last instance that missed the deadline.
+  If no deadline was ever missed, it will have value ``c_InstanceHandle_Unknown``.
+
+.. warning::
+
+    Currently this status is not supported and will be implemented in future releases.
+    As a result, trying to access this status will return ``NOT_SUPPORTED``
+    and the corresponding listener will never be called.
+
+
+.. _dds_layer_core_status_requestedIncompatibleQosStatus:
+
+RequestedIncompatibleQosStatus
+------------------------------
+
+This status changes every time the :ref:`dds_layer_subscriber_dataReader` finds a
+:ref:`dds_layer_publisher_dataWriter` that matches the :ref:`dds_layer_topic_topic` and has
+a common partition, but with a QoS that is incompatible with the one defined on the
+:ref:`dds_layer_subscriber_dataReader`.
+
+List of status data members:
+
++--------------------------+-----------------------------------------+
+| Data Member Name         | Type                                    |
++==========================+=========================================+
+| total_count              | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| total_count_change       | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| last_policy_id           | ``uint32_t``                            |
++--------------------------+-----------------------------------------+
+| policies                 | ``std::vector<QosPolicyCount>``         |
++--------------------------+-----------------------------------------+
+
+* **total_count**: Total cumulative count of :ref:`DataWriters<dds_layer_publisher_dataWriter>` found
+  matching the :ref:`dds_layer_topic_topic` and with a common partition, but with a QoS that is incompatible
+  with the one defined on the :ref:`dds_layer_subscriber_dataReader`.
+
+* **total_count_change**: The change in **total_count** since
+  the last time |on_requested_incompatible_qos| was called or the status was read.
+  It can only have zero or positive values.
+
+* **last_policy_id**: The policy ID of one of the policies that was found to be incompatible with the
+  current :ref:`dds_layer_subscriber_dataReader`.
+  If more than one policy happens to be incompatible, only one of them will be reported in this member.
```

Comment:
I know we don't have an implementation so it's no big deal, but which is the criteria to report the policy? Is it just the last one checked? It's not very clear in this description

Reply:
The standard says nothing, so it is implementation defined.
And, as you say, we have no implementation yet. I wanted to clarify the criteria, but we still have none. And just saying 'the one that is reported is up to the implementation' seems wrong, we are describing OUR implementation, not the standard.

Therefore, I left it as you found. If you have any better idea, I'm all ears.